src/client/definition/EClientEmbeddedSDKActions.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * The actions provided by the AppClientEmbeddedSDK.
+ */
+export enum EClientEmbeddedSDKActions {
```

Comment:
What is the reason for the `E` in front of the name of this enum?

Reply:
Just to let others know that it is an **E**numeration type.

Reply:
We currently aren't doing that for other enums, so my vote would be to leave that off (https://rocketchat.github.io/Rocket.Chat.Apps-engine/enums/roomtype.html).

Reply:
Thank you, I see. Will change it.

Reply:
Resolved in https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/165/commits/f76385ca193c5b20653e189497e63357965f3e8e